Automattic/wp-calypso
# Hosting onboarding: Create developer specific site setup checklist

Related to https://github.com/Automattic/dotcom-forge/issues/2629

## Proposed Changes

* Add checklist task for developer accounts.

## Testing Instructions

- Apply patch D113245-code
- Select AT site
- Verify dev checklist is shown for a developer account.
- Complete each task and verify tasks are checked,



![image](https://github.com/Automattic/wp-calypso/assets/47489215/644b346c-b89c-4116-ae6e-b78e318f125e)


## Pre-merge Checklist
